src/rpc/blockchain.cpp

```diff
@@ -1034,6 +1034,10 @@ UniValue mempoolInfoToJSON()
     ret.push_back(Pair("maxmempool", (int64_t)maxmempool));
     ret.push_back(Pair("mempoolminfee", ValueFromAmount(mempool.GetMinFee(maxmempool).GetFeePerK())));
 
+    char buf[50];
+    sprintf(buf, "%.2f", mempool.TransactionsPerSecond());
```

Comment:
Actually, and sorry for the annoyance, but isn't this a perfect case for `strprintf` from `tinyformat.h`?

That would avoid dealing with writing to fixed char buffers altogether.

Reply:
Actually just tried that, the problem is it returns 
"tps"  "0.00"

rather than 

"tps" 0.00

seems a small issue but the extra quotes could cause a little data collection parsing headache for those wanting to get the info from time to time and track the data.

Reply:
Ok, found a way, took the output of strprintf and did a lexical_cast<double> which get's rid of the ""

Reply:
Seems to be a pretty roundabout way to deal with this but I guess it works :D